uniffi_bindgen/src/bindings/kotlin/templates/NamespaceLibraryTemplate.kt

```diff
@@ -55,20 +55,65 @@ internal open class {{ ffi_struct.name()|ffi_struct_name }}(
 {%- endmatch %}
 {%- endfor %}
 
+
+{%- macro decl_kotlin_functions(func_list) -%}
```

Comment:
Called from `UniffiLib` (as usual) but this time **without the checksum functions** - for Kotlin only. Other language bindings are not affected.

And also called from new `interface UniffiLibChecksums`, **with checksum functions only**